src/main/java/com/codegym/shopyy/model/Comment.java

```diff
@@ -0,0 +1,32 @@
+package com.codegym.shopyy.model;
+
+import com.fasterxml.jackson.annotation.JsonIgnoreProperties;
+import jakarta.persistence.*;
```

Comment:
use import-specific library instead of import .*

Reply:
resolve